lib/ruby_smb/dcerpc/p_cont_list_t.rb

```diff
@@ -0,0 +1,37 @@
+module RubySMB
+  module Dcerpc
+    # The Bind PDU as defined in
+    # [The bind PDU](http://pubs.opengroup.org/onlinepubs/9629399/chap12.htm#tagcjh_17_06_04_03)
```

Comment:
These references should be moved to the `Bind` definition and I would add this instead:
```suggestion
    # The presentation context list and its element as defined in
    # [Connection-oriented PDU Data Types - Declarations](https://pubs.opengroup.org/onlinepubs/9629399/chap12.htm#tagcjh_17_06_03_01)
```